src/main/java/seedu/club/commons/util/StringUtil.java

```diff
@@ -65,4 +65,22 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Returns true if {@code s} has length less than or equal {@code length}
+     * {@code s} will have leading and trailing whitespaces trimmed before comparing length.
+     * <br>examples:<pre>
+     *      *       hasLessThanOrEqualLength("hello world", 10) == true
+     *      *       hasLessThanOrEqualLength("hello", 3) == true
+     *      *       hasLessThanOrEqualLength("hello  ", 6) == false
+     *      *       hasLessThanOrEqualLength("hello", 6) == false
```

Comment:
wait am i tripping or do these have wrong output
```suggestion
     *      *       hasLessThanOrEqualLength("hello world", 10) == true
     *      *       hasLessThanOrEqualLength("hello", 3) == false
     *      *       hasLessThanOrEqualLength("hello  ", 6) == true
     *      *       hasLessThanOrEqualLength("hello", 6) == true
```